packages/presentation/presentation_publishing/interfaces/has_library_transforms.ts

```diff
@@ -59,8 +105,6 @@ export const apiHasLibraryTransforms = <StateT extends object = object>(
       typeof (unknownApi as HasLibraryTransforms<StateT>).canLinkToLibrary === 'function' &&
       typeof (unknownApi as HasLibraryTransforms<StateT>).canUnlinkFromLibrary === 'function' &&
       typeof (unknownApi as HasLibraryTransforms<StateT>).saveToLibrary === 'function' &&
-      typeof (unknownApi as HasLibraryTransforms<StateT>).getByReferenceState === 'function' &&
```

Comment:
Agree. I have been testing with my Links embeddable and this is one change that I had to revert. Otherwise inline library transforms erroneously trigger [this conditional](https://github.com/elastic/kibana/pull/182523/files#diff-b097e776bd25da07c086725fd5c0db814fae2da0c7a1638f773c59095541eb00R101-R103).

Reply:
Resolved in [541b84b](https://github.com/elastic/kibana/pull/182523/commits/541b84bdee44c2d4018f7c12720770bba34681b4)